Exercise_D/assets/style/index.css

```diff
@@ -0,0 +1,76 @@
+/* 
+This css sheet is linked to practice.html
+You can style tags, classes (.) or ids (#) from this file.
+*/
+
+.head{
+    background:linear-gradient(rgba(0,0,0,0.5), rgba(0,0,0,0.5)), url(/Exercise_D/assets/images/flowershop.jpg);
```

Comment:
Nice use of linear-gradient to darken you image nice one! Great use of responsive size for you height